test/Regex/control_character_escapes.js

```diff
@@ -0,0 +1,201 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+function matchRegExp(str, regexpLiteral, expectedResult)
+{
+    matchResultLiteral = str.match(regexpLiteral);
+    errorMsgBase = "Expected result of match between string: '" + str + "' and regular expression: " + regexpLiteral + " to be " + 
+                    expectedResult + " but was "
+
+    actualResultLiteral = matchResultLiteral == null ? null : matchResultLiteral[0];
+    assert.areEqual(expectedResult, actualResultLiteral, errorMsgBase + actualResultLiteral); 
+    
+    regexpConstructor = new RegExp(regexpLiteral);
+    matchResultConstructor = str.match(regexpConstructor);
+
+    actualResultConstructor = matchResultConstructor == null ? null : matchResultConstructor[0];
+    assert.areEqual(expectedResult, actualResultConstructor, errorMsgBase + actualResultConstructor); 
+}
+
+var tests = [
+    {
+        name : "Control characters followed by a word character ([A-Za-z0-9_])",
+        body : function () 
+        {
+            re = /[\c6]+/; //'6' = ascii x36

```

Comment:
>\c6 [](start = 19, length = 3)

can you also include in the comments which byte \cX parses to? (Or if not a byte, what it is expected to parse to?)